plugins/TimingHardwareManagerPDII.cpp

```diff
@@ -49,14 +50,15 @@ TimingHardwareManagerPDII::conf(const nlohmann::json& conf_data)
   register_hsi_hw_commands_for_design();
 
   auto conf_params = conf_data.get<timinghardwaremanagerpdi::ConfParams>();
```

Comment:
Where does this conf_params come from, haven't we got rid of the JSON configuration?